### PR TITLE
add filter comment

### PR DESCRIPTION
**Pull request by JaofourthsOhappy**
add filter comment and user can delete menu that user create

### CODE REVIEW

File: foods/admin.py [hunk 1]
```diff
@@ -1,10 +1,11 @@
 from django.contrib import admin
 
-from .models import Menu, FoodOfDay
+from .models import Menu, FoodOfDay,Comment
```
Comment (jaybjackie) [inline foods/admin.py:3]: Please leave a blank space before `Comment`

File: foods/views.py [hunk 2]
```diff
@@ -11,11 +11,13 @@
 from .forms import RegisterForm
 from django.contrib import messages
 import datetime
-from foods.models import Menu ,FoodOfDay, MenuRating,CookBook,Comment,Reply
+from foods.models import Menu ,FoodOfDay, MenuRating,CookBook,Comment
```
Comment (jaybjackie) [inline foods/views.py:14]:
```suggestion
from foods.models import Menu ,FoodOfDay, MenuRating, CookBook, Comment
```

File: foods/views.py [hunk 3]
```diff
@@ -264,6 +269,7 @@ def cook_create(request):
     if request.method == "POST":
         name = request.POST.get("title")
         des = request.POST.get("description")
```
Comment (jaybjackie) [inline foods/views.py:271]: For variable name, I for sure our member will understand the intention of variable.
But I would like to convince everyone to write descriptive name for variable.
Please change the variable name to be more specific for good practice.
Cheers!

Comment (jaybjackie) [reply foods/views.py:271]: included 
`des`
`Ing`
`totalcal` I suggest to be something like `total_cal` or `total_calories`
`fatcal` suggested as above
`sugargrams` also as above

**Review by jaybjackie — COMMENTED** [review]
There are a minor suggest change. Please have a look.

**Comment by jaybjackie** [comment]
I have test this PR.
image

It's works! If you have time, decorate the inputs to a bit right hand so, it will be easier to look.

You doing great!! Keep working and make sure you have a good rest.
Thank you for your hardwood Krub 😸

**Comment by zevenfox** [comment]
> I have test this PR. image
> 
> It's works! If you have time, decorate the inputs to a bit right hand so, it will be easier to look. I for sure you still working on further step of this page. Take your time.
> 
> You doing great!! Keep working and make sure you have a good rest. Thank you for your hardwood Krub 😸

It work for me too! Then let merge it. 
@JaofourthsOhappy  don't forget to fix some of code that me and Benz reviews and we also need to decorate it too. Thx for ur hard work krub🤩

**Pull request merged**